plugins/source/aws/docs/tutorials/multi_account.md

```diff
@@ -0,0 +1,144 @@
+
+   ```yaml
+   org:
+     member_role_name: OrganizationAccountAccessRole
+     admin_account:
+       local_profile: <Named-Profile>
+   ```
+
+   3. Assuming a role in admin account using credentials in the shared configuration or credentials file:
+
+   ```yaml
+       org:
+         member_role_name: OrganizationAccountAccessRole
+         admin_account:
+           local_profile: <Named-Profile>
+           role_arn: arn:aws:iam::<ACCOUNT_ID>:role/<ROLE_NAME>
+
+           // Optional. Specify the name of the session
+           // role_session_name: ""
+
+           // Optional. Specify the ExternalID if required for trust policy
+           // external_id: ""
+   ```
+
+3. Optional. If the trust policy configured for the member accounts requires different credentials than you configured in the previous step, then you can specify the credentials to use in the `member_trusted_principal` block
+
+```yaml
+org:
+  member_role_name: OrganizationAccountAccessRole
+  admin_account:
+    local_profile: <Named-Profile-Admin>
+  member_trusted_principal:
+    local_profile: <Named-Profile-Member>
+  organization_units:
+    - ou-<ID-1>
+    - ou-<ID-2>
+```
+
+4. Optional. If you want to specify specific Organizational Units to fetch from you can add them to the `organization_units` list.
+
+```yaml
+org:
+  member_role_name: OrganizationAccountAccessRole
+  admin_account:
+    local_profile: <Named-Profile-Admin>
+  organization_units:
+    - ou-<ID-1>
+    - ou-<ID-2>
+```
+
+**_note: If you specify an OU, CloudQuery will not traverse child OUs_**
+
+#### Arguments for Org block
+
+- `organization_units` **(Optional)** - List of Organizational Units that CloudQuery should use to source accounts from
+- `admin_account` **(Optional)** - Configuration on how to grab credentials from an Admin account
+- `member_trusted_principal` **(Optional)** - Configuration on how to specify the principle to use in order to assume a role in the member accounts
+- `member_role_name` **(Required)** - Role name that CloudQuery should use to assume a role in the member account from the admin account. Note: This is not a full ARN, it is just the name
+- `member_role_session_name` **(Optional)** - Override the default Session name.
+- `member_external_id` **(Optional)** - Specify an ExternalID for use in the trust policy
+- `member_regions` **(Optional)** - Limit fetching resources within this specific account to only these regions. This will override any regions specified in the provider block. You can specify all regions by using the `*` character as the only argument in the array
+
+## Multi Account- Specific Accounts
+
+CloudQuery can fetch from multiple accounts in parallel by using AssumeRole (You will need to use credentials that can AssumeRole to all other specified account. Following is an example configuration:
+
+```yaml
+providers:
+  - name: aws
+    resources:
+      - "*"
+    configuration:
```

Comment:
This bit still needs to be updated for the new config format

Reply:
I'll do that in follow-up PR there was too much work in this one. and we might still change some config fields in the upcoming aws PR.